ipydb/magic.py

```diff
@@ -185,24 +193,27 @@ def sql(self, args='', cell=None):
             params = self.shell.user_ns.get(args.params, {})
         if args.multiparams:
             multiparams = self.shell.user_ns.get(args.multiparams, [])
-        result = self.ipydb.execute(sql, params=params,
+        cursor = self.ipydb.execute(sql, params=params,
                                     multiparams=multiparams)
+        
+        if not cursor:
+            return None
+        if not cursor.returns_rows:
+            s = 's' if cursor.rowcount != 1 else ''
+            print("%i row%s affected" % (cursor.rowcount, s))
+
+        if args.pandas:
+            return self.ipydb.build_dataframe(cursor)
         if args.ret:
-            return result
-        if result and result.returns_rows:
-            if args.single:
-                self.ipydb.render_result(
-                    PivotResultSet(result), paginate=False, filepath=args.file)
-            else:
-                self.ipydb.render_result(
-                    result, paginate=not bool(args.file), filepath=args.file)
-        elif result and not result.returns_rows:
-            # XXX: do all drivers support this?
-            s = 's' if result.rowcount != 1 else ''
-            print("%i row%s affected" % (result.rowcount, s))
-    sql.__description__ = 'Run an sql statement against ' \
```

Comment:
yeah, for this snippet:  

```
 100          m = getattr(SqlMagics, magic, None)
  101          if m:
  102:             if hasattr(m, '__description__'):
  103:                 doc = m.__description__
  104              else:
  105                  doc = getattr(m, '__doc__', '')
```

I was wondering to simplify it as bellow, for the docstring is enough. anyway, let's put this in next fix. I will recover the ' __description__' in this pr.

```
 100          m = getattr(SqlMagics, magic, None)
  101          if m:
  105                  doc = getattr(m, '__doc__', '')
```